controller/api/public/grpc_server.go

```diff
@@ -39,6 +39,7 @@ type grpcServer struct {
 	discoveryClient       discoveryPb.DiscoveryClient
 	k8sAPI                *k8s.API
 	controllerNamespace   string
+	clusterDomain         string
```

Comment:
This shouldn't be necessary. See comment on reading the cluster domain from the `linkerd-config` config map.

Reply:
Removed